<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuscarCita</class>
 <widget class="QMainWindow" name="BuscarCita">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buscar cita ID</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1081</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Citas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por ID cita</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="InputIdCita">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por ID paciente</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="InputIdPaciente">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>190</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por fecha de cita</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>260</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AÃ±o</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>260</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>260</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BuscarFecha">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>310</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>310</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hora</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>310</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minutos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BuscarIdPaciente">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>190</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BuscarIdCita">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tablaCita">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>621</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
*{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
	font: 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>ID Cita</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Paciente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Paterno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Materno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de cita</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="inputA">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2021</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2022</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2025</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2026</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2027</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2028</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2029</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2030</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="inputM">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>260</y>
       <width>73</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-01-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-02-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-03-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-04-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-05-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-06-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-07-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-08-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-09-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-10-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-11-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-12-</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="inputD">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>260</y>
       <width>73</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="inputH">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>310</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> 23</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="inputMin">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>310</y>
       <width>73</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:45</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>MenÃº</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
